spec/index.md

```diff
@@ -60,86 +60,56 @@ Fiscal Data Package specifies the *form* for fiscal data and offers a standardiz
 * Mapping the raw "physical" model, as represented by columns in the data files, to a standardized "logical" model based around basic fiscal concepts: amounts spent, suppliers, administrative and functional classifications etc
 * Progressive enhancement of data via a range of *recommended*, but not *required* metadata, in order to establish a clear path for data providers to enhance data quality, and to address new use cases going forward.
 
-
-# Background
-
-This proposal assumes some familiarity with fiscal data - e.g. budgets, spending etc - as this is the data we are structuring and describing.
-
-Often, this data takes the form of rows in a spreadsheet or database with each row describing some kind of expenditure or receipt of money. The data can get considerably more complex but keep this simple model in mind for what follows.
-
-```
-+--------+------+------------+------------+
```

Comment:
Generally excllent and my only question was re removing this table: thought this could be useful in having all readers sharing the same mental model

I am not sure on this point so happy to have a push to remove since smaller is better here!